.gitignore

```diff
@@ -3,6 +3,8 @@
 
 .env
 
+src/test/__snapshots__/
```

Comment:
This does not look correct.  Can you explain why we are not including the snapshots?